save-backend/src/test/kotlin/org/cqfn/save/backend/controller/TestInitializeControllerTest.kt

```diff
@@ -67,4 +69,17 @@ class TestInitializeControllerTest {
 
         assertTrue(databaseData.any { it.id == test.id && it.expectedFilePath == test.expectedFilePath })
     }
+
+    @Test
+    fun checkServiceData() {
```

Comment:
Can we insert 21 executions into DB here to test if batching works as expected?

Reply:
Tested